src/plugins/plan/pluginSpec.js

```diff
@@ -104,7 +105,7 @@ describe('the plugin', function () {
         ];
         let planView;
 
-        beforeEach(() => {
+        beforeAll(() => {
```

Comment:
See above. I think we should continue to use beforeEach here. Every test should start in the exact same state. In this case, a test might modify `planDomainObject`. Resetting that object before each test will prevent that issue.